packages/shared/src/store/storage/storageActions.ts

```diff
@@ -38,6 +55,7 @@ export const uploadAsset = async (asset: ImagePickerAsset, isWeb = false) => {
   try {
     logger.log('resizing asset', asset.uri);
     let resizedAsset = asset;
+    const originalMimeType = asset.mimeType;
     // avoid resizing gifs
     if (!asset.mimeType?.includes('gif')) {
       resizedAsset = await manipulateAsync(
```

Comment:
doesn't this automatically convert to JPEG? if I read this correctly I think that we'd convert something to JPEG and then later on try to tack on the original mime which would give it an incorrect extension.